ibmcloud

dostaric opened the pull request:
Pour le compte de l'itération 2, 

J'ai fait la configuration pour le déploiement de notre application sur CLOUD d'IBM. 
Les fichiers .travis.yml et manifest.yml ont été modifiés pour prendre en compte cette fonctionnalité.

Commit:
Implementing configuration for ibmcloud deployement

Commit:
implementing on file .travis.yml the possibility to deploy on branch feat/ibmcloud

Commit:
implementing on file manifest.yml the possibility to deploy on branch feat/ibmcloud

Commit:
implementing on file manifest.yml the possibility to deploy on branch feat/ibmcloud

oliviercailloux commented:
Merci d’ajouter à votre README l’URL de base sur laquelle le projet est déployé, ainsi qu’une URL qui envoie directement vers un servlet GET pour pouvoir observer que ça marche.

dostaric commented:
ok bien recu

Commit:
implementing servet for testing deployement on ibmcloud

Commit:
implementing servet for testing deployement on ibmcloud

Commit:
adding url in the ReadMe file for accessing online to the application deployed on ibm cloud

dostaric committed:
Update README.adoc

adding url in the ReadMe file for accessing online to the application deployed on ibm cloud

dostaric commented:
Bonjour Monsieur ,

Urls rajoutés au fichier ReadMe comme demandé.
Vous pouvez acceder au get via ce lien en dessous.

https://dauphine-open-datas.eu-gb.mybluemix.net/resource/home
